qubesadmin/tests/mock_app.py

```diff
@@ -474,6 +549,16 @@ def __init__(self, qapp):
                     b'2\x00QubesNoSuchPropertyError\x00\x00No such property\x00'
                 continue
 
+        for property_name, value in DEFAULT_DOM0_PROPERTIES.items():
```

Comment:
This (admin.property.Get) should be about global properties. Dom0 properties you handle above already.